packages/terra-list/docs/guides/MultiSelectList.md

```diff
@@ -1,6 +1,6 @@
 # Terra List - Implementing a Multi Select List
 
-In previous versions of the terra-list a multi select style list could be created as either a controlled version of SelectableList or uncontrolled version of MultiSelectList.  These implementations suffered from inflexibility and performance concerns. Going foward terra-list is more granular, though this puts more reponsibility on the consumer to properly update their list items with the appropriate state. The following is a guide to addressing those concerns in your implementation.
+The terra-list implementation requires controlled state if selections are required. As a result selections are applied at child items generation from HOC state. The following guide show you how to implement that state within a multiple items selection variant of list, as well as implement the additional narrowing requirements of a max selection for items.
```

Comment:
Just a few typos in this paragraph.

- I think the second sentence is trying to say 'As a result, selections are applied when child items are generated from HOC state.
- The following guide will show you

Reply:
Updated.